### ISSUE TITLE
Class Diagram

### ISSUE THREAD

**Issue by RomanBoegli** [TKone7/lightread#3]
@RomanBoegli @TKone7

**Comment by RomanBoegli** [comment]
@TKone7 I uploaded a first draft version. We should think about how we will interact with our objects and to what extend they should manage DB-interactions (_to be discussed_).

_btw: I generated the UML diagram with an open source tool called [UMLet](https://www.umlet.com/)_

**Comment by TKone7** [comment]
Hi
Thanks for your draft. I worked on it and extended it in the following ways.
- Database interactions should be handled by the corresponding [Data Access Objects](https://github.com/webengfhnw/WE-CRM#data-access-objects-dao). I created `ContentDAO`, `UserDAO`, `PaymentDAO` and `WithdrawalDAO`.
- Additionally, I added some basic Service-Interfaces and their implementations according to the [reference project](https://github.com/webengfhnw/WE-CRM#service-interfaces-and-implementations).
I suggest we talk through it on the phone.
Regards
